src/app/components/typeFilter/TypeIconList/type/flying.tsx

```
@@ -1,30 +1,30 @@
+			height={size}
+			viewBox="0 0 512 512"
+			fill="none"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<svg
+				width="512"
+				height="512"
+				viewBox="0 0 512 512"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)